crates/holochain_zome_types/Cargo.toml

```diff
@@ -88,6 +83,8 @@ fuzzing = [
   "holochain_serialized_bytes/fuzzing",
   "holo_hash/fuzzing",
   "kitsune_p2p_timestamp/fuzzing",
+  "rand",
+  "fixt",
```

Comment:
`cargo test` on this crate didn't work. It was relying on some other crate enabling the `fixtures` feature which happens when you run `cargo test` from the root but not from the crate directory.